picture-in-picture/css-pseudo-class.html

```diff
@@ -0,0 +1,70 @@
+---
+feature_name: CSS pseudo-class ":picture-in-picture"
+chrome_version: 76
+feature_id: 6259534815297536
+check_min_version: true
+---
+
+<h3>Background</h3>
+<p>
+The <a href="https://wicg.github.io/picture-in-picture/">Picture-in-Picture
+API</a> allows websites to create a floating video window that is always on top
+of other windows so that users may continue consuming media while they interact
+with other sites or applications on their device. In Chrome 76, a CSS
+pseudo-class is added to help web developers customize the media player when
+videos enter and exit Picture-in-Picture.
+</p>
+<p>
+Credits: Media files are © copyright Blender Foundation | <a href="http://www.blender.org">www.blender.org</a>.
+</p>
+
+<style>
+  button { margin: 4px 0; }
+</style>
+
+{% capture css %}
+#video {
+  display: block;
+  width: 100%;
+  will-change: opacity;
+}
+
+#video:not([controls]):picture-in-picture {
```

Comment:
is :not([controls]) needed?

Reply:
It is not needed per se, but it is good practice to not hide video with native controls in Picture-in-Picture as this is the only way for now to navigate in video.